app/scripts/seller-transform.js

```diff
@@ -192,6 +225,16 @@ var sellerTransform = (function(_, relatedEntityTransform, commonTransforms) {
                 newData.date = infoBuckets(aggs.phone.timeline.buckets,'date','city',['publisher','mentions'],{'publisher':'Info'});
             }
             return newData;
+        },
+        locationTimeline: function(data){
```

Comment:
Unit tests for this new transform should be added to app/test/client-config-seller-test.html.